### PR TITLE
update readme.md

### PR DESCRIPTION
**Pull request by indekkusu-era**
Updated a descriptive documentation on the Workflow of 4dm media centre article writing, please check if anything is missing

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+
+Then click `Add File > Upload Files`
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009838181480415313/unknown.png)
+
+Then you drop your markdown file or choose from your directory and write a **Commit Changes**
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009838767114309693/unknown.png)
+
+Then click **Commit Changes**.
```
Comment (DannyPX) [inline readme.md]: This workflow doesn't actually require VSC then. Is VSC really recommended then?
Would prefer to have 2 sections one with Notion and one with VSC then. (Would also be nice to show how to export in Notion)

File: readme.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+Click **Create Pull Request** and it will bring you to this page
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009840139146641478/unknown.png)
+
+Edit title and description of your Pull Request as you like, then click **Create Pull Request**. Copy the URL and send it to 4dm Media Centre Group Chat.
+
+### Reviews
+
+In order to review a Pull Request, you click into a Pull Request URL and go to "Files changed", Click "Review Changes" and add your comment
```
Comment (DannyPX) [inline readme.md]:
```suggestion
In order to review a Pull Request from a different author, you click into a Pull Request URL and go to "Files changed", Click "Review Changes" and add your comment
```

File: readme.md [hunk 3]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009842057034084352/unknown.png)
+
+If you think this article still needs changes, click "Comment" or "Request Changes". Or if you think it's ready-to-go click "Approve".
+
+Normally, it's like ranking a map, you need two BNs to approve your map in order to be ranked. This one you need at least two contributors to Approve your pull request in order to merge into the main branch (and ready to be shown).
```
Comment (DannyPX) [inline readme.md:136]: Maybe add that you can request from specific people as well?

File: readme.md [hunk 4]
```diff
@@ -1,3 +1,79 @@
...
+
+### Creating File / Uploading File
+
+After finishing your Article Markdown, go to the folder (stories/interviews/opinions) and upload the file based on category you wrote
+
+For example, if you create a new story, you upload the story by clicking `stories` folder
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009837843469844542/unknown.png)
+
+Then click `Add File > Upload Files`
```
Comment (DannyPX) [inline readme.md]:
```suggestion
Then click **Add File** and then **Upload Files** in the dropdown menu.
```

File: readme.md [hunk 5]
```diff
@@ -1,3 +1,79 @@
-# 4 Digit MWC Media Centre Articles Repository
+# 4 Digit MWC Media Centre Articles
 
-hi
+This is a documentation for 4 Digit MWC Media Centre Workflow via Github
```
Comment (DannyPX) [inline readme.md]:
```suggestion
This is a documentation for 4 Digit MWC Media Centre Workflow via Github.
```

File: readme.md [hunk 6]
```diff
@@ -1,3 +1,79 @@
...
+
+### Github
+
+In order to get access to **Media Centre** and **4dm Repositories** in general, create a [Github](https://github.com/) account, then contact me (HowToPlayLN) with the Github profile link in order to be added into a [4digitmwc Organization](https://github.com/4digitmwc).
+
+### Create Branch
+
+You will see many repositories, but the one we're going to use to write the articles is [media-centre-articles](https://github.com/4digitmwc/media-centre-articles).
+
+Create a separate branch by clicking at the branch icon. The branch name must be in format `yourname/<updates>` (example below)
```
Comment (DannyPX) [inline readme.md]:
```suggestion
Create a separate branch by clicking at the branch icon. The branch name must be in format `yourname/<updates>` (example below).
```

File: readme.md [hunk 7]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009835163632545915/unknown.png)
+
+Then click "Create branch".
+
+### Creating File / Uploading File
+
+After finishing your Article Markdown, go to the folder (stories/interviews/opinions) and upload the file based on category you wrote
```
Comment (DannyPX) [inline readme.md]:
```suggestion
After finishing your Article Markdown, go to the folder (stories/interviews/opinions) and upload the file based on category you wrote.
```

File: readme.md [hunk 8]
```diff
@@ -1,3 +1,79 @@
...
+
+For example, if you create a new story, you upload the story by clicking `stories` folder
```
Comment (DannyPX) [inline readme.md]:
```suggestion
For example, if you create a new story, you upload the story by clicking `stories` folder.
```

File: readme.md [hunk 9]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009837843469844542/unknown.png)
+
+Then click `Add File > Upload Files`
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009838181480415313/unknown.png)
+
+Then you drop your markdown file or choose from your directory and write a **Commit Changes**
```
Comment (DannyPX) [inline readme.md]:
```suggestion
Then you drop your markdown file or choose from your directory and write a commit message.
```

File: readme.md [hunk 10]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009838767114309693/unknown.png)
+
+Then click **Commit Changes**.
+
+### Pull Requests
+
+After you added new articles or update them, you need to do **Pull Request** for articles to be reviewed. You start by clicking **Pull Requests**
```
Comment (DannyPX) [inline readme.md]:
```suggestion
After you added new articles or update them, you need to do **Pull Request** for articles to be reviewed. You start by clicking **Pull Requests**.
```

File: readme.md [hunk 11]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009839539717677216/unknown.png)
+
+Then, click **New Pull Request**, then change the `compare:` to your branch (below for example)
```
Comment (DannyPX) [inline readme.md]:
```suggestion
Then, click **New Pull Request**, then change the `compare:` to your branch (below for example).
```

File: readme.md [hunk 12]
```diff
@@ -1,3 +1,79 @@
...
+
+![](https://cdn.discordapp.com/attachments/546525809440194560/1009839898469089321/unknown.png)
+
+Click **Create Pull Request** and it will bring you to this page
```
Comment (DannyPX) [inline readme.md]:
```suggestion
Click **Create Pull Request** and it will bring you to this page.
```

File: readme.md (hunk 2 above)
Comment (DannyPX) [inline readme.md]:
```suggestion
In order to review a Pull Request, you click into a Pull Request URL and go to "Files changed", Click "Review Changes" and add your comment.
```

File: readme.md (hunk 4 above)
Comment (DannyPX) [reply readme.md]: Would check consistency on this throughout the whole article.

**Review by DannyPX — CHANGES_REQUESTED** [review]
Probably not too important for now. But instead of saying click/clicking, use select/selecting.

File: readme.md (hunk 2 above)
Comment (DannyPX) [reply readme.md]:
```suggestion
In order to review a Pull Request, you click into a Pull Request URL and go to "Files changed", click "Review Changes" and add your comment
```

File: readme.md (hunk 1 above)
Comment (indekkusu-era) [reply readme.md]: VSC is just another option to edit markdown, and I am planning to provide resources or write a brief documentation for these

File: readme.md (hunk 3 above)
Comment (indekkusu-era) [reply readme.md:136]: uhh can you elaborate more of this ? from what I understand, you want me to add the documentation about review request feature on github right ?

Comment (DannyPX) [reply readme.md:136]: Yes, since its more likely (in this case) that we actually review something that is requested.

File: readme.md (hunk 1 above)
Comment (DannyPX) [reply readme.md]: Will it be in this readme or something separate?

Comment (indekkusu-era) [reply readme.md]: planning to be included in readme

File: readme.md (hunk 3 above)
Comment (indekkusu-era) [reply readme.md:136]: alright

File: readme.md (hunk 1 above)
Comment (indekkusu-era) [reply readme.md]: done

File: readme.md (hunk 3 above)
Comment (indekkusu-era) [reply readme.md:136]: done

**Review by polytetralhx — APPROVED** [review]
hi, seems all good to go on my end, reverse MTGL

**Pull request merged**